.github/workflows/tags_job.yml

```diff
@@ -55,3 +61,42 @@ jobs:
           echo ${workspace}
     outputs:
       environment_workspace_name: ${{ steps.name_workspace.outputs.name }}
+
+  pull_tags:
+    if : inputs.changes_detected == 'false'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1.7.0
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::311462405659:role/modernising-lpa-github-actions-ssm-get-parameter
+          role-duration-seconds: 900
+          role-session-name: GithubActionsSSMGetParameter
+      - name: Pull production environment tag
+        id: pull_production_tag
+        run: |
+          value=$( aws ssm get-parameter --name '/modernising-lpa/container-version/preproduction' | jq -r .Parameter.Value )
```

Comment:
pull production tag but gets preproduction?

Reply:
we're not writing a production tag at the moment because it's not on the path to live.